src/App.jsx

```diff
@@ -14,33 +16,81 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = 'alphabetSort';
+const SORT_BY_LENGTH = 'goodsLength';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function sortGoods(goods, { sortField, isReversed }) {
+  const newSortGoogs = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField === SORT_BY_ALPHABET) {
+    newSortGoogs.sort((a, b) => a.localeCompare(b));
+  } else if (sortField === SORT_BY_LENGTH) {
+    newSortGoogs.sort((a, b) => a.length - b.length);
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  return isReversed ? newSortGoogs.reverse() : newSortGoogs;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState(null);
```

Comment:
The initial state for `sortField` is set to `null`. This might cause issues when sorting since the `sortGoods` function expects a valid sort field. Consider setting a default sorting method.